lib/matplotlib/backends/qt_compat.py

```diff
@@ -41,13 +46,13 @@
 # Otherwise, check the QT_API environment variable (from Enthought).  This can
 # only override the binding, not the backend (in other words, we check that the
 # requested backend actually matches).
-elif rcParams["backend"] == "Qt5Agg":
+elif rcParams["backend"] in ["Qt5Agg", "Qt5Cairo"]:
```

Comment:
would `.startswith("Qt5")` be better here?

Reply:
Doesn't really help with mplcairo... and I think we can revisit this if a third rendering backend ever show up :)
(In a sense the whole Qt4/5 setup is a bit weird, basically what we really have is just a Qt{Agg,Cair} backend and the 4/5 part just sets the default for QT_API.)
Anyways I don't feel strongly either way.